doc/source/text.rst

```diff
@@ -306,20 +306,20 @@ The same alignment can be used when ``others`` is a ``DataFrame``:
 Concatenating a Series and many objects into a Series
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-All one-dimensional list-likes can be arbitrarily combined in a list-like container (including iterators, ``dict``-views, etc.):
+An arbitrary combination of ``Series``, ``Index`` and ``np.ndarray`` (1-dimensional) can be passed in a list-like container (including iterators, ``dict``-views, etc.):
 
 .. ipython:: python
 
     s
     u
-    s.str.cat([u.values, ['A', 'B', 'C', 'D'], map(str, u.index)], na_rep='-')
+    s.str.cat([u.values, u, u.index.astype(str)], na_rep='-')
```

Comment:
can you save these in another variable, rather than doing the combination in-line. its easier to see it printed out (with comments)

Reply:
What do you mean by "comments"?